Add CHANGELOG and new README (#426)
bufbuild/vscode-buf: README.md

# Buf for Visual Studio Code

> The [VS Code Buf extension][vs-code-marketplace] provides rich support for [Protocol Buffers][protobuf] using the [Buf][buf] development environment.

Reviewer: Might suggest being a bit more direct with the problem we're solving. Something like...

> The [VS Code Buf extension][vs-code-marketplace] helps you work with [Protocol Buffers][protobuf] files in a much more intuitive way, adding smart syntax highlighting, navigation, formatting, and Buf command support.

## Features

- **Code navigation** - Go-to definition and documentation insets for `.proto` symbols.
- **Syntax highlighting** - Protobuf specific color and styling of code.
- **Code editing** - Formatting via `buf format` and annotations based on `buf lint` and `buf breaking`.
- **Buf command support** - Execution of `buf` CLI commands via the [Command Palette][command-palette].

> ![Preview features](./preview.gif)

Reviewer: Left a general note above, but just to add here - this Gif is really hard to see. I'm happy to redo it for you if I can get up an running over here.

Author: Yes please, would happily take this offer once we get the development environment stuff sorted out!

## Getting Started

[Install the latest version via the VS Code marketplace][vs-code-marketplace].

By default, the extension will use your locally-installed version of `buf` on your system

> `$PATH`. However, you are not required to have `buf` installed, the extension can manage and

Reviewer 2: ```suggestion
`$PATH`. However, you don't have to install `buf`—the extension can manage and
```
